CONTRIBUTING.md

```diff
@@ -77,9 +77,14 @@ In the sample configuration, the Hive connector is mounted in the `hive` catalog
 
 To build the Presto docs, see the [docs README](presto-docs/README.md).
 
-### Building the Web UI
-
-The Presto Web UI is composed of several React components and is written in JSX and ES6. This source code is compiled and packaged into browser-compatible JavaScript, which is then checked in to the Presto source code (in the `dist` folder). You must have [Node.js](https://nodejs.org/en/download/) and [Yarn](https://yarnpkg.com/en/) installed to execute these commands. To update this folder after making changes, simply run:
+### Building the Presto Console
+
+The Presto Console is composed of several React components and is written in JSX and ES6. This source code is
+compiled and packaged into browser-compatible JavaScript and stored in the `presto-main/src/main/resources/webapp/dist`
+folder. You must have [Node.js](https://nodejs.org/en/download/) and [Yarn](https://yarnpkg.com/en/) installed to
```

Comment:
```suggestion
folder. 

You must have [Node.js](https://nodejs.org/en/download/) and [Yarn](https://yarnpkg.com/en/) installed to
```
Suggest a line break here to emphasize the prerequisites, to make it less likely for the reader to overlook them.